exporter/signalfxexporter/README.md

```diff
@@ -89,7 +89,7 @@ The following configuration options can also be configured:
 that are allowed to be used as a dimension key in addition to alphanumeric 
 characters. Each nonalphanumeric dimension key character that isn't in this string 
 will be replaced with a `_`.
-- `max_connections` (default = 100):  The maximum number of idle HTTP connection the exporter can keep open.
+- `max_connections` (default = 100):  The maximum number of idle HTTP connection the exporter can keep open. Deprecated. Prefer using [HTTP settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/README.md) instead.
```

Comment:
I would rather reword it to insist on not using it. We will need to remove deprecated fields eventually.
```suggestion
- `max_connections` (default = 100):  The maximum number of idle HTTP connections the exporter can keep open. Deprecated: use `max_idle_conns` or `max_idle_conns_per_host` instead. See [HTTP settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/README.md) for more info.
```